### ISSUE TITLE
comment #8

### ISSUE THREAD

**Issue by Iqbal-khan07** [vap38/CS435_Project2#8]
Does this astar work? It needs a lot of comments since poor name of variables makes it hard to follow the code; 
an example
`for v in tempNode.vertices:`
it can be 
`
neighbours = tempNode.vertices
for neighbor in neighbours
     .......`
It makes it much more clear what the algorithm is trying to do.
https://github.com/vap38/CS435_Project2/blob/c69f244b05592c79d575ca071ef73700042a28e5/aStar/GridGraph.py#L121

**Comment by vap38** [comment]
Commit 527173e
